### ISSUE TITLE
Incorrect camera settings assignement

### ISSUE THREAD

**Issue by avaver** [godlikemouse/jquery.cube.threejs.js#2]
Hi, great job with the cube.
There is a small typo when setting up camera
```
        //setup camera
        _camera.position.y = options.camera.x;
        _camera.position.x = options.camera.y;
        _camera.position.z = options.camera.z;
```

x and y positions are swapped.

**Comment by godlikemouse** [comment]
Thanks for finding this, I'll fix it ASAP.

**Comment by godlikemouse** [comment]
Fixed in commit 4e64b452d7d21d828578a8e04aa5139321b251ab
